zebedee-cms/src/main/java/com/github/onsdigital/zebedee/filters/AuthenticationFilter.java

```diff
@@ -89,12 +170,24 @@ public boolean filter(HttpServletRequest request, HttpServletResponse response)
         return result;
     }
 
+    private void unauthorisedRequest(HttpServletResponse response, String errorMessage) throws IOException {
+        response.setContentType("application/json");
+        response.setStatus(HttpStatus.UNAUTHORIZED_401);
+        Serialiser.serialise(response, errorMessage);
+    }
```

Comment:
If you do something like this (unAuth response by default to error response if unsuccessful):
```
    private void unauthorisedRequest(HttpServletResponse response, String errorMessage) {
        try {
            response.setContentType("application/json");
            response.setStatus(HttpStatus.UNAUTHORIZED_401);
            Serialiser.serialise(response, errorMessage);
        } catch (IOException ex) {
            error(response);
        }
    }
```
Then you won't have to catch the thrown IOException multiple times in the `verifyAndStoreAccessToken`. You can then remove the try catch blocks and it will more straight forward.

Reply:
👍